Packs/PrismaCloud/Triggers/Trigger_-_AWS_CloudTrail_Misconfiguration.json

```diff
@@ -0,0 +1,33 @@
+{
+  "trigger_id": "fc37d4039563569b21b5bebec0a92500",
+  "playbook_id": "Prisma Cloud Remediation - AWS CloudTrail Misconfiguration v2",
+  "suggestion_reason": "Recommended for AWS CloudTrail Misconfiguration alerts",
+  "description": "This trigger is responsible for handling AWS CloudTrail Misconfiguration alerts",
+  "trigger_name": "AWS CloudTrail Misconfiguration",
+  "alerts_filter": {
+    "filter": {
```

Comment:
We can use the "alert type" as the trigger since the logic is being done at the classifier level. Same goes for all of the triggers in this PR.
This will lead to a single place of config for future changes instead of having 2 places.